miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleService.java

```diff
@@ -226,6 +233,10 @@ public Long create(Sample sample) throws IOException {
           throw new IOException(e.getMessage(), e);
         }
         validateHierarchy(detailed);
+
+        if (detailed.getDetailedQcStatus() != null) {
```

Comment:
Hmm, looks like that code is already present in `loadChildEntities`. I'll delete it from `create` and `update`.